src/CommunityToolkit.Aspire.Hosting.ActiveMQ/ActiveMQBuilderExtensions.cs

```diff
@@ -0,0 +1,111 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Utils;
+using CommunityToolkit.Aspire.Hosting.ActiveMQ;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding ActiveMQ resources to an <see cref="IDistributedApplicationBuilder"/>.
+/// </summary>
+public static class ActiveMQBuilderExtensions
+{
+    /// <summary>
+    /// Adds a ActiveMQ container to the application model.
+    /// </summary>
+    /// <remarks>
+    /// The default image and tag are "apache/activemq-classic" and "6.1.0".
+    /// </remarks>
+    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
+    /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency.</param>
+    /// <param name="userName">The parameter used to provide the user name for the ActiveMQ resource. If <see langword="null"/> a default value will be used.</param>
+    /// <param name="password">The parameter used to provide the password for the ActiveMQ resource. If <see langword="null"/> a random password will be generated.</param>
+    /// <param name="port">The host port that the underlying container is bound to when running locally.</param>
+    /// <param name="scheme">The scheme of the endpoint, e.g. tcp or activemq (for masstransit). Defaults to tcp.</param>
+    /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
+    public static IResourceBuilder<ActiveMQServerResource> AddActiveMQ(this IDistributedApplicationBuilder builder,
+        string name,
```

Comment:
```suggestion
        [ResourceName] string name,
```

The `ResourceNameAttribute` is used to enforce the naming convention via an analyzer.